### ISSUE TITLE
What qualifies as paying their speakers?

### ISSUE THREAD

**Issue by bitemyapp** [abiggerhammer/conferencesthatpaytheirspeakers.org#1]
Sorry to be a pest, but was wondering what fits the criteria. Is it enough that they defray some of the cost of travel? All of it? Travel + payment for taking the time?

**Comment by abiggerhammer** [comment]
The standard I'm looking for is travel, hotel, and honorarium for the talk.

If that's succinct enough, I could just put that in the subtitle.

**Comment by TheSweetKat** [comment]
Could there be separate categories for those that just provide honoraria, vs those that pay travel and lodging? (On the current list, DEF CON pays only an honorarium or extra badges; Black Hat covers travel.)

**Comment by abiggerhammer** [comment]
Good point, and having made the change I mentioned above, DEF CON is now actually wrong. I'll go ahead and split those out.

**Comment by bitemyapp** [comment]
Having three buckets along the lines described seems like they could be useful.
1. Partial compensation of expenses
2. All expenses
3. All expenses + honorarium

Thanks for explaining!
